internal/command/docker.go

```diff
@@ -38,6 +62,34 @@ type Options struct {
 	Debug        bool     // Debug if true enable log verbosity
 }
 
+// initIsPodman returns true if docker is an alias to podman (e.g. via podman-docker).
+func initIsPodman() bool {
+	// read the "docker" executable to check if it's a podman wrapper
+
+	if os.Getenv("FYNECROSS_FORCE_PODMAN") == "true" {
```

Comment:
I see the pro to force an engine. If you agree I'd move to  a fyne-cross flag like `--engine=<type>` where the possibile values are: `auto`, `docker`, `podman` (default to auto-detect)

Reply:
It's what I wanted to do initially - I will make the adaptations.